projects/plugins/jetpack/modules/masterbar/featured-post-thumbnail/class-featured-post-thumbnail.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Featured Post Thumbnail Support on WP.com.
```

Comment:
Can we change that to "Featured Image - post-thumbnail Support", removing WP.com also since it will run for Jetpack self hosted sites too.

Reply:
I've changed the comment.